macros/miscs/startup.cfg

```diff
@@ -103,6 +108,14 @@ gcode:
     {action_respond_info(parameters.output)}
 
 
+[gcode_macro _INIT_ERCF]
+gcode:
+    {% set ercf_enabled = printer["gcode_macro _USER_VARIABLES"].ercf_enabled %}
+    {% if ercf_enabled and printer.ercf.enabled|int != 1 %}
```

Comment:
As it's already checked if `ercf_enabled` is true in the `KLIPPAIN_STARTUP` macro, you can remove it from here and just use instead:
```
{% if printer.ercf.enabled|int != 1 %}
```

Reply:
can we replace `{% if printer.ercf.enabled|int != 1 %}` by `{% if printer.ercf.enabled %}`?
this variable introduce by HH is boolean (True or False)